pyaedt/modules/CableModeling.py

```diff
@@ -109,6 +110,7 @@ def __init__(self, app, json_file_name=None, working_dir=None):
         if json_file_name:
             self._init_from_json(json_file_name)
 
+    @pyaedt_function_handler()
```

Comment:
Why do we need the handler function decorator if no modifications are made to the methods' signatures? No args modified it seems.

Reply:
@MaxJPRey this class would need a refactor so I wasn't sure to touch it or not. I'd like to get rid of all the try/except blocks and use the decorator to catch any exception if anything fails but I'd like to that in a different PR.

Reply:
OK, got it. Thanks @gmalinve.